content/developer/reference/backend/views.rst

```diff
@@ -2527,6 +2527,132 @@ For instance if ``foo`` and ``bar`` refer to two groupbys
 
 has the effect to activate first ``bar`` then ``foo``.
 
+View customizations
+-------------------
+
+As seen in :ref:`Customizing an existing view <reference/js_class>` we can create a custom version
+of a generic view. Here we will present some of these views:
+
+Settings View
+~~~~~~~~~~~~~
+
+The settings view is a customization of the form view. It's used to centralize all the settings of Odoo.
+This view differs from a generic form view because it has a search panel and a sidebar.
+
+To avoid complecity, simplify and standardize the settings archs, a small DSL exclusively for the
+settings was created. This new DSL introduces 3 tags: ``app``, ``block`` and ``setting``.
+
+.. rst-class:: o-definition-list
+
+``app``
+  The ``app`` tag is used to declare the application on the settings view. It creates an entry with
+  its logo on the sidebar of the view. It also acts as delimiter when searching.
+
+  Syntax:
+
+  .. code-block:: xml
+
+    <app string="CRM" name="crm">
+    ...
+    </app>
+
+  Parameters:
+
+  .. rst-class:: o-definition-list
+
+  ``string``
+    The "display" name of the application.
+  ``name``
+    The technical name of the application (the name of the module).
+  ``logo`` (optional)
+    The relative path to the logo. If not set, the logo is created using the ``name`` parameter : ``/{name}/static/description/icon.png``.
+
+``block``
+  The ``block`` tag is used to declare a group of settings. This group can have a title and a description/help.
+
+  Syntax:
+
+  .. code-block:: xml
+
+    <block title="Title of group Bar">
+    ...
+    </block>
+
+  Parameters:
+
+  .. rst-class:: o-definition-list
+
+  ``title`` (optional)
+    The title of the block of settings (the old h2), you can perform research on its text.
```

Comment:
in the doc, there should be no notion of "old" (unlike in the commit, where you explain what changes).